common/djangoapps/django_comment_common/models.py

```diff
@@ -109,6 +109,20 @@ def has_permission(self, permission):
 
         return self.permissions.filter(name=permission).exists()
 
+    @staticmethod
+    def user_has_role_for_course(user, course_id, role_names):
```

Comment:
i'm not entirely sure what this file is, but why not put this function in something like student/models.py?